container/cassandra/docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+version: "3.9"
+services:
+  cassandra1:
+    restart: always
+    build:
+      context: .
+    ports:
```

Comment:
Does this one need to seed `cassandra1` as well? Or does the default of localhost work in this context?